<ui version="4.0" >
 <class>FotoLab_Class</class>
 <widget class="QMainWindow" name="FotoLab_Class" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FotoLab</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../Resources/Icons/icons.qrc" >
    <normaloff>:/icons/process.png</normaloff>:/icons/process.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="ImageLab::ImageView" name="graphicsView" />
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="separator" />
   <addaction name="actionClose" />
   <addaction name="separator" />
   <addaction name="actionOpen" />
   <addaction name="actionSaveAs" />
   <addaction name="separator" />
   <addaction name="actionZoomIn" />
   <addaction name="actionZoomOut" />
   <addaction name="actionZoomNormal" />
   <addaction name="separator" />
   <addaction name="actionReload" />
   <addaction name="separator" />
  </widget>
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>667</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionReload" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="separator" />
    <addaction name="actionZoomIn" />
    <addaction name="actionZoomOut" />
    <addaction name="actionZoomNormal" />
    <addaction name="separator" />
    <addaction name="actionProcess" />
    <addaction name="actionProperties" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionHelpAbout" />
    <addaction name="separator" />
    <addaction name="actionHelpContents" />
    <addaction name="actionHelpFaq" />
    <addaction name="separator" />
    <addaction name="actionAboutQt" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QToolBar" name="toolBar_2" >
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionAreaColor" />
   <addaction name="actionFillArea" />
   <addaction name="actionCut" />
   <addaction name="actionEdges" />
   <addaction name="actionLines" />
   <addaction name="actionDrawArea" />
  </widget>
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/edit.png</normaloff>:/icons/edit.png</iconset>
   </property>
   <property name="text" >
    <string>Cut</string>
   </property>
  </action>
  <action name="actionZoomIn" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoomNormal" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/zoom-original.png</normaloff>:/icons/zoom-original.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom Normal</string>
   </property>
  </action>
  <action name="actionProcess" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/process.png</normaloff>:/icons/process.png</iconset>
   </property>
   <property name="text" >
    <string>Process</string>
   </property>
  </action>
  <action name="actionHelpAbout" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/help-about.png</normaloff>:/icons/help-about.png</iconset>
   </property>
   <property name="text" >
    <string>Help About</string>
   </property>
  </action>
  <action name="actionHelpContents" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/help-contents.png</normaloff>:/icons/help-contents.png</iconset>
   </property>
   <property name="text" >
    <string>Help Contents</string>
   </property>
  </action>
  <action name="actionHelpFaq" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/help-faq.png</normaloff>:/icons/help-faq.png</iconset>
   </property>
   <property name="text" >
    <string>Help Faq</string>
   </property>
  </action>
  <action name="actionAboutQt" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/qt.png</normaloff>:/icons/qt.png</iconset>
   </property>
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAreaColor" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/color.png</normaloff>:/icons/color.png</iconset>
   </property>
   <property name="text" >
    <string>Change Color of Area</string>
   </property>
  </action>
  <action name="actionFillArea" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/fill.png</normaloff>:/icons/fill.png</iconset>
   </property>
   <property name="text" >
    <string>Fill Area</string>
   </property>
  </action>
  <action name="actionEdges" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/process.png</normaloff>:/icons/process.png</iconset>
   </property>
   <property name="text" >
    <string>Edges</string>
   </property>
  </action>
  <action name="actionReload" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/reload.png</normaloff>:/icons/reload.png</iconset>
   </property>
   <property name="text" >
    <string>reload Image</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/document-save-as.png</normaloff>:/icons/document-save-as.png</iconset>
   </property>
   <property name="text" >
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionProperties" >
   <property name="text" >
    <string>Properties...</string>
   </property>
  </action>
  <action name="actionLines" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/lines.png</normaloff>:/icons/lines.png</iconset>
   </property>
   <property name="text" >
    <string>Lines</string>
   </property>
  </action>
  <action name="actionDrawArea" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../Resources/Icons/icons.qrc" >
     <normaloff>:/icons/pen.png</normaloff>:/icons/pen.png</iconset>
   </property>
   <property name="text" >
    <string>drawArea</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageLab::ImageView</class>
   <extends>QGraphicsView</extends>
   <header>../gui/imageview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/Icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
